STYLE_GUIDE.md

```diff
@@ -1,57 +1,68 @@
 TODO:: Outstanding Topics:
+
 - shared vs. core components
 - Style management (BEM/CSS modules/etc.)
 - Storybook
 
 ## Intro
+
 This is a collection of choices/preferences/decisions around how to build within the Zero ecosystem. It is a collection of guidance intended to help keep things maintainable and provide some consistency across the project as well as related projects. It is also a repository of decisions that have been made including reasoning where appropriate, to reduce the need for repeat discussions.
 
 ## Components
+
 The responsibility of components is to display data and accept user input. Business logic including data manipulation does not belong at the component level.
 
 ### Function vs. Class Components
+
 Function and class components both have their place. Function components can be used for simple components that don't require a lot of logic around how the data is displayed. For anything that requires a bit more complexity, a class is generally going to be more appropriate.
 
 #### Function Components
+
 The general wisdom around functions in software development also applies to function components. They should be very short, accept minimal input props, and return clear and consistent markup. If you can't figure out what a function component requires, renders, and is doing at a glance, then it is too long.
 
 - A function component should be no more than 15-20 lines excluding markup.
 - The markup should be a single block, and should be restricted to the return statement.
 - The markup should also have very minimal logic contained within it.
 
 #### Class Components
+
 The general wisdom around classes in software development also applies to class components.
 
 ### Hooks
+
 Hooks are a useful tool when their strengths and limitations are understood. They provide a means to add some state and a bit of additional logic to a function component without having to convert it to a class or break out multiple components. For simple components this can result in something that is a bit more straightforward and easier to understand than using a class, and can increase development speed for small changes. As with function components, hooks should be short, accept minimal inputs, and should be "grokable" at a glance.
 
 - Hooks should be a maximum of 15-20 lines.
-- Hooks cannot be nested. eg. you cannot call a *custom* hook from another hook. You can use built-in hooks within a custom hook.
+- Hooks cannot be nested. eg. you cannot call a _custom_ hook from another hook. You can use built-in hooks within a custom hook.
```

Comment:
curious, what's the difference b/w using `*<x>*` & `_<x>_`?

Reply:
My understanding is that they behave the same. The auto formatter made that change.